### PR TITLE
Formalize-yoneda-embedding-2

### PR DESCRIPTION
**Pull request by cesarbm03**
The proof of remak 9.4 as given in RS17. If this is correct, I can dualize and include the contravariant case.

### CODE REVIEW

**Comment by emilyriehl** [comment]
Hi @cesarbm03 does this PR overwrite the previous one? It seems like this one might make further edits on the definitions you've included there.

If so, maybe cancel that pull request (though you can first look at the comments I just left) and then we can review this one?

**Comment by cesarbm03** [comment]
Yes, this one finishes the proof. I branched from formalize-yoneda, so I thought this one #103  would supersede #100 by default. I going to make the modifications to this pull request according to your comments on the previous one and cancel it afterwards.
I just saw the issue on the style this afternoon, I was unaware of it until today. I'll be implementing from now on, so hopefully my submissions are cleaner.

**Comment by cesarbm03** [comment]
Yes, absolutely. I will let you know once it is ready. I'll keep an eye on the Zulip stream for suggestions.

**Comment by cesarbm03** [comment]
Regarding the implementation of the changes, names and the formatting, I found that I can either edit the file on Github or do the changes locally and then force commits to the branch with
`$ git commit -m "These changes are in response to PR comments"
$ git push -f origin HEAD` 

What's the recommended or best practice for this?

**Comment by jonweinb** [comment]
Thanks @cesarbm03! Do you have any insights on this @fizruk, @fredrik-bakke?

**Comment by fizruk** [comment]
> Regarding the implementation of the changes, names and the formatting, I found that I can either edit the file on Github or do the changes locally and then force commits to the branch with `$ git commit -m "These changes are in response to PR comments"` `$ git push -f origin HEAD`
> 
> What's the recommended or best practice for this?

I will note a few things:
1. Since your PR is not merged yet, it is perfectly fine to push to it.
2. Editing files on GitHub is okay, but local development is preferrable.
3. However, I do not think you should "force push", regular push should be enough.
4. Also, you should push to a specific branch, e.g. `formalize-yoneda-embedding-2`, not `HEAD`.
5. I also see that this PR is not for `main`, but for `formalize-yoneda-embedding` branch. Usually, there is no need to do a PR for another PR (and also this PR does not have automated checks because of this) and you can just work on your changes in one branch (and one PR) and we will merge them into `main`.

I would suggest to merge this PR (it will merge into `formalize-yoneda-embedding`, not `main`) and continue working and discussing in #100.

File: src/simplicial-hott/09-yoneda.rzk.md [hunk 1]
```diff
@@ -432,29 +429,65 @@ We prove the Yoneda embedding.
...
-        ( is-segal-A)
-        ( a')) φ
+      ( is-covariant-representable-is-segal A is-segal-A a') φ
+
+#def htpy-yoneda-phi uses (funext)
+  ( A : U)
+  ( is-segal-A : is-segal A)
+  ( a a' : A)
+  ( φ : ( x : A) → ( hom A a x → hom A a' x) )
+  : (x : A) → ( ( yoneda-embedding A is-segal-A a a' ) ( ( evid A a ( hom A a')) φ )) x = φ x
```
Comment (fredrik-bakke) [inline src/simplicial-hott/09-yoneda.rzk.md:439]:
```suggestion
  : (x : A) →
    yoneda-embedding A is-segal-A a a' (evid A a (hom A a') φ) x = φ x
```
There's a couple of things to point out here:
- Our conventions only stipulate placing spaces after opening parentheses that appear at the start of a line. The reason for that is how this makes your code nicely align with other indented subexpressions.
- Overly parenthesizing expressions is detrimental to readability. The general exceptions to this is when your expressions involve lambdas or commas, which can generally be confusing to parse without the added parentheses.
- We try to keep code within an 80-character line length limit.

Comment (fredrik-bakke) [reply src/simplicial-hott/09-yoneda.rzk.md:439]: Also, there is no need to insert a space before a closing parenthesis.

**Comment by cesarbm03** [comment]
Thank you for the comments @fizruk and @fredrik-bakke. Once this is merged in #100 I'll make the changes there.

**Review by fredrik-bakke — APPROVED** [review]

**Comment by fredrik-bakke** [comment]
Sorry to hold you up!

**Pull request merged**